libs/rtemodel/src/RteProject.cpp

```diff
@@ -449,14 +448,28 @@ void RteProject::InitFileInstance(RteFileInstance* fi, RteFile* f, int index, Rt
   fi->SetRemoved(false);
   string absPath = fi->GetAbsolutePath();
   bool bExists = RteFsUtils::Exists(absPath);
-  if (!bExists) {
-    UpdateFileInstance(fi, f, false, false);
-  } else {
+  if (bExists) {
     fi->AddAttribute("version", savedVersion, false);
   }
-  UpdateConfigFileBackups(fi, f);
```

Comment:
> 3\. Restructuring InitFileInstance() and adding WriteInstanceFiles() : all cses should be considered, not only loading cprj file

Were there any specific cases you had in mind that wasn't covered?

Reply:
The call to `UpdateConfigFileBackups` creates files in the RTE folder, so that's why I break it out and put in the new `WriteInstanceFiles` that can be called separately based on whether RTE files should be updated or not.